src/phoebus_driver.cpp

```diff
@@ -351,6 +352,28 @@ TaskCollection PhoebusDriver::RungeKuttaStage(const int stage) {
     }
   }
 
+  // Extra per-step user work
+  TaskRegion &sync_region_5 = tc.AddRegion(num_partitions);
+  PARTHENON_REQUIRE(num_partitions == 1,
+                    "Reductions don't work for multiple partitions?");
```

Comment:
This is what I was saying above, it's because you have multiple `MPI_AllReduce`s per rank. If you use the Parthenon reduction machinery this problem goes way. Or you can make sure this task region *only* gets called with 1 partition.

Reply:
Yep, will fix.

Reply:
Yes good catch thanks, it's removed.